src/test/static/js-behaviors.js

```diff
@@ -33,4 +36,4 @@ Really.Really.Deep.Behavior = [
 @polymerBehavior
 */
 CustomBehaviorList =
-    [SimpleBehavior, CustomNamedBehavior, Really.Really.Deep.Behavior];
+    [SimpleBehavior, AwesomeBehavior, Really.Really.Deep.Behavior];
```

Comment:
Yes. So this code is incorrect unless there was some other context that assigned CustomNamedBehavior to the AwesomeBehavior global, but that's what the jsdoc is telling us happens. Fortunately this is just a test fixture, this code is never executed :)